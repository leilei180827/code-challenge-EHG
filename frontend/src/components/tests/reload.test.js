import React from 'react';
import renderer from 'react-test-renderer';
import Reload from '../reload';

test('reload render correctly', () => {

  const component = renderer.create(
    <Reload></Reload>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
