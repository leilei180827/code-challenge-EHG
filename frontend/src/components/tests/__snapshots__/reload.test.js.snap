// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reload render correctly 1`] = `
<div
  id="reload"
>
  <div
    className="text"
  >
    Unfortunately, something bad happened,Please try to reload it
  </div>
  <div
    className="reload-icon"
  >
    <img
      alt="refresh"
      className="reload-image"
      onClick={[Function]}
      src="refresh.png"
    />
  </div>
</div>
`;
